e2e-tests/src/data/constants.ts

```diff
@@ -0,0 +1,28 @@
+export const Study = {
```

Comment:
External file for strings. Hardcode string in tests and lib are bad test practices.